iotdb/src/main/java/org/apache/iotdb/db/utils/SyncUtils.java

```diff
@@ -104,54 +101,49 @@ public static boolean verifyIPSegment(String ipWhiteList, String ipAddress) {
 
   /**
    * Verify IP address with IP segment.
-   *
-   * @param ipSegment
-   * @param ipAddress
-   * @param subnetMark
-   * @return
    */
   private static boolean verifyIP(String ipSegment, String ipAddress, int subnetMark) {
-    String ipSegmentBinary = "";
-    String ipAddressBinary = "";
-    String[] ipSplits = ipSegment.split("\\.");
+    String ipSegmentBinary;
+    String ipAddressBinary;
+    String[] ipSplits = ipSegment.split(IP_SEPARATOR);
     DecimalFormat df = new DecimalFormat("00000000");
     StringBuilder ipSegmentBuilder = new StringBuilder();
     for (String IPsplit : ipSplits) {
-      ipSegmentBuilder.append(String.valueOf(df.format(
-              Integer.parseInt(Integer.toBinaryString(Integer.parseInt(IPsplit))))));
+      ipSegmentBuilder.append(df.format(
+          Integer.parseInt(Integer.toBinaryString(Integer.parseInt(IPsplit)))));
     }
     ipSegmentBinary = ipSegmentBuilder.toString();
     ipSegmentBinary = ipSegmentBinary.substring(0, subnetMark);
-    ipSplits = ipAddress.split("\\.");
+    ipSplits = ipAddress.split(IP_SEPARATOR);
     StringBuilder ipAddressBuilder = new StringBuilder();
     for (String IPsplit : ipSplits) {
-      ipAddressBuilder.append(String.valueOf(df.format(
-              Integer.parseInt(Integer.toBinaryString(Integer.parseInt(IPsplit))))));
+      ipAddressBuilder.append(df.format(
+          Integer.parseInt(Integer.toBinaryString(Integer.parseInt(IPsplit)))));
     }
     ipAddressBinary = ipAddressBuilder.toString();
     ipAddressBinary = ipAddressBinary.substring(0, subnetMark);
     return ipAddressBinary.equals(ipSegmentBinary);
   }
 
-  public static void deleteFile(File file) throws IOException {
+  /**
+   * Remove all files under this folder recursively
+   *
+   * @param file folder file
+   */
+  public static void deleteFile(File file) {
```

Comment:
Just as a note: FileUtils in apache common provides the same function.

Reply:
I've replaced my implementation with this util